### PR TITLE
fix error when gradient is None

### PR DESCRIPTION
**Pull request by blairhan**
When I trained ResNet in FP16, it gave an error since some gradients' values are None and cannot be divided by the loss_scale.

### CODE REVIEW

**Comment by blairhan** [comment]
I checked it and I am the only author. @googlebot

**Comment by blairhan** [comment]
My git config user.email had a type-o and i fixed it. @googlebot

**Comment by robieta** [comment]
What gradients are None?

**Comment by blairhan** [comment]
@robieta sorry for the late reply. 

I ran the model using this script https://github.com/mlperf/training/blob/master/image_classification/tensorflow/run.sh with fp16 setting on and it shows the error. There are some issues on this like https://github.com/mlperf/training/issues/230. As far as I know, tf.gradients() returns None instead of zero and that may cause the error for loss_scale division.

**Comment by jaeyounkim** [comment]
Deprecated
